[EN/FR] Add/Cleanup `History of osu! 2017` (#6769)
ppy/osu-wiki: wiki/History_of_osu!/2017/fr.md

Liens :

- [New Featured Artist: Ben Briggs](https://osu.ppy.sh/home/news/2017-02-08-new-featured-artist-ben-briggs)
- [Release 0.0.3 · ppy/osu](https://github.com/ppy/osu/releases/tag/v0.0.3)

## Mars

Le Monthly Beatmapping Contest revolving pour osu!catch a commencé. [LukHash](https://osu.ppy.sh/beatmaps/artists/12) a été annoncé en tant que nouvel Featured Artist. Les votes pour Aspire 2017 ont commencé. Spotlights a été annoncé pour co-exister avec les chartes de classement. Le 9ème concours de fanart sur le thème printanier a ouvert.

Liens :

- [Monthly Beatmapping Contest March 2017: osu!catch](https://osu.ppy.sh/home/news/2017-03-01-monthly-beatmapping-contest-march-2017-osucatch)
- [New Featured Artist: LukHash](https://osu.ppy.sh/home/news/2017-03-03-new-featured-artist-lukhash)
- [Aspire 2017 - Voting Now Open!](https://osu.ppy.sh/home/news/2017-03-13-aspire-2017-voting-now-open)
- [Introducing to you: Spotlights](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights)
- [Fanart Contest \#9: Spring](https://osu.ppy.sh/home/news/2017-03-21-fanart-contest-9-spring)

## Avril

> Le Monthly Beatmapping Contest \#13 a conclu avec [RyoKazuka](https://osu.ppy.sh/users/6258586) à la première place. Aspire 2017 : Stage One s'est conclu par la première place de [ProfessionalBox](https://osu.ppy.sh/users/3250792). Les votes pour le concours de fanart printanier ont ouvert. Le 100ème(?) numéro du osu!weekly est sorti. La osu!taiko World Cup 2017 s'est conclue par la victoire du Japon sur la Taïwan. Avant que le mois d'avril se termine, le concours de fanart printanier ont permis à [Evaine](https://osu.ppy.sh/users/7295733) de prendre la première place.

Reviewer: ```suggestion
Le Monthly Beatmapping Contest \#13 s'est conclu avec [RyoKazuka](https://osu.ppy.sh/users/6258586) à la première place. Aspire 2017 : Stage One s'est conclu par la première place de [ProfessionalBox](https://osu.ppy.sh/users/3250792). Les votes pour le concours de fanart printanier ont été ouverts. Le 100ème(?) numéro du osu!weekly est sorti. La osu!taiko World Cup 2017 s'est conclue par la victoire du Japon face à Taïwan. Avant que le mois d'avril se termine, le concours de fanart printanier a permis à [Evaine](https://osu.ppy.sh/users/7295733) de prendre la première place.
```

Liens :

- [Monthly Beatmapping Contest \#13 Results](https://osu.ppy.sh/home/news/2017-04-02-monthly-beatmapping-contest-13-results)
- [Aspire 2017 Stage One Concludes](https://osu.ppy.sh/home/news/2017-04-03-aspire-2017-stage-one-concludes)
- [Spring Fan Art Contest Voting Now Open](https://osu.ppy.sh/home/news/2017-04-11-spring-fan-art-contest-voting-now-open)
- [osu!weekly \#100](https://osu.ppy.sh/home/news/2017-04-15-osuweekly-100)
- [osu!taiko World Cup 2017 Results](https://osu.ppy.sh/home/news/2017-04-22-osutaiko-world-cup-2017-results)

## Mai

> La osu!Catch World Cup 2017 a été ouverte aux inscriptions. [Kuba Oms](https://osu.ppy.sh/beatmaps/artists/13) a été annoncé en temps que nouveau Featured Artists pour osu!. osu! a annoncé être à l'Anime Expo 2017. Après plus d'un an, la nouvelle mascotte osu!taiko a été annoncée. [Mocha](/wiki/Mascots#mocha) est entrée sur scène avec [pippi](/wiki/Mascots#pippi), [Yuzu](/wiki/Mascots#yuzu), et [Maria](/wiki/Mascots#mani-&-mari). Plus tard, [Rin / Function Phantom](https://osu.ppy.sh/beatmaps/artists/14) a été annoncé pour rejoindre les Featured Artists d'osu!. Les votes pour osu!mani et osu!catch beatmapping contest ont tous deux ouverts.

Reviewer: ```suggestion
La osu!Catch World Cup 2017 a été ouverte aux inscriptions. [Kuba Oms](https://osu.ppy.sh/beatmaps/artists/13) a été annoncé en tant que nouveau Featured Artists pour osu!. osu! a annoncé être à l'Anime Expo 2017. Après plus d'un an, la nouvelle mascotte osu!taiko a été annoncée. [Mocha](/wiki/Mascots#mocha) est entrée sur scène avec [pippi](/wiki/Mascots#pippi), [Yuzu](/wiki/Mascots#yuzu), et [Maria](/wiki/Mascots#mani-&-mari). Plus tard, [Rin / Function Phantom](https://osu.ppy.sh/beatmaps/artists/14) a été annoncé en tant que nouveau featured artist d'osu!. Les votes ont été ouverts pour le Beatmapping Contest des modes osu!mania et osu!catch.
```

Liens :

- [osu!catch World Cup 2017 - Registrations now open!](https://osu.ppy.sh/home/news/2017-05-03-osucatch-world-cup-2017-registrations-now-open)
- [osu! at anime expo 2017](https://osu.ppy.sh/home/news/2017-05-09-osu-at-anime-expo-2017)
- [The new osu!taiko mascot is here!](https://osu.ppy.sh/home/news/2017-05-25-the-new-osutaiko-mascot-is-here)
- [New Featured Artist: Rin / Function Phantom](https://osu.ppy.sh/home/news/2017-05-30-new-featured-artist-rin-function-phantom)
- [Monthly Beatmapping Contest Voting Now Open: osu!catch and osu!mania](https://osu.ppy.sh/home/news/2017-06-01-monthly-beatmapping-contest-voting-now-open)

## Juin

> La team osu! participera à l'Anime Expo pour les 10 prochaines années d'osu!. Le 15-07-2017, le menu principal/playfield a été remplacé par 4 nouvelles images génériques. Les images de playfield changent quand on change de musique. LE support permettant d'activer ou de désactiver les seasonal backgrounds toute l'année, ou de temps en temps a été rajouté dans les [Options](/wiki/Client/Options). Pendant le mois, dans le Discord osu!dev, il y a eu des discussions autour de la mort du osu!weekly. À partir de ce moment-là chaque post officiel et communautaire sera postés quand il sera prêt, plutôt que de le faire chaque semaine. Le Mocha in Summer Fanart Contest a commencé.

Reviewer: ```suggestion
L'équipe d'osu! participera à l'Anime Expo pour les 10 prochaines années d'osu!. Le 15 juin 2017, le menu principal/playfield a été remplacé par 4 nouvelles images génériques. Les images de playfield changent quand on change de musique. La possibilité d'activer ou de désactiver les seasonal backgrounds toute l'année, ou de temps en temps a été rajoutée dans les [Options](/wiki/Client/Options). Pendant le mois, dans le Discord osu!dev, il y a eu des discussions autour de la mort du osu!weekly. À partir de ce moment-là, chaque post officiel et communautaire sera posté quand il sera prêt, plutôt que de le faire chaque semaine. Le Mocha in Summer Fanart Contest a commencé.
```

Liens :

- [2017.609.0 – ppy blog](https://blog.ppy.sh/2017.609.0/)
- [2017.717.0 – ppy blog](https://blog.ppy.sh/2017.717.0/)
- [main menu background changes](https://osu.ppy.sh/community/forums/topics/606931)
- [Mocha in Summer Fanart Contest Begins!](https://osu.ppy.sh/home/news/2017-06-15-mocha-fanart-contest-now-open)

## Juillet

> Les inscriptions pour la osu!mania 4K World Cup ont ouvert. Les votes pour le Mocha in Summer Fanart Contest ont ouvert. La osu!catch World Cup 2017 s'est conclue par la victoire de la Corée du Sud sur la Chine. les enregistrements pour la Community Mentorship Program Fall 2017 ont ouverts. Au dernier jour de juillet, la Mocha in Summer fanart contest s'est conclue avec [2Su](https://osu.ppy.sh/users/6598966) à la première place.

Reviewer: ```suggestion
Les inscriptions pour la osu!mania 4K World Cup sont ouverts. Les votes pour le Mocha in Summer Fanart Contest sont ouverts. La osu!catch World Cup 2017 s'est conclue par la victoire de la Corée du Sud sur la Chine. Les inscriptions pour le Community Mentorship Program Fall 2017 sont ouvertes. Au dernier jour de juillet, le Mocha in Summer Fanart Contest s'est conclu avec [2Su](https://osu.ppy.sh/users/6598966) à la première place.
```

Liens :

- [osu!mania 4K World Cup 2017 - Registrations now open!](https://osu.ppy.sh/home/news/2017-07-10-osumania-4k-world-cup-2017-registrations-now-open)
- [Mocha in Summer Fanart Contest Voting Open](https://osu.ppy.sh/home/news/2017-07-11-mocha-in-summer-fanart-contest-voting-open)
- [osu!catch World Cup 2017 - Results](https://osu.ppy.sh/home/news/2017-07-14-osucatch-world-cup-2017-results)
- [Community Mentorship Program Fall 2017 Signups Now Open](https://osu.ppy.sh/home/news/2017-07-18-community-mentorship-program-fall-2017-signups-now-open)

## Août

> [Wisp X](https://osu.ppy.sh/beatmaps/artists/16) et [OISHII](https://osu.ppy.sh/beatmaps/artists/17) ont été annoncés en temps que nouveaux Featured Artists. Les auditions pour la osu!idol 2017 ont ouvert. Le osu! coffee Hour a été introduit en tant que podcast. Un concours de fanart pour un nouveau look de [Yuzu's](/wiki/Mascots#yuzu) a ouvert.

Reviewer: ```suggestion
[Wisp X](https://osu.ppy.sh/beatmaps/artists/16) et [OISHII](https://osu.ppy.sh/beatmaps/artists/17) ont été annoncés en tant que nouveaux featured artists. Les auditions pour la osu!idol 2017 ont ouvert. La osu! Coffee Hour a été introduite en tant que podcast. Un concours de fanart pour un nouveau look de [Yuzu](/wiki/Mascots#yuzu) a été ouvert.
```

Liens :

- [w Featured Artist: Wisp X](https://osu.ppy.sh/home/news/2017-08-10-new-featured-artist-wisp-x)
- [osu!idol 2017 Auditions Now Open](https://osu.ppy.sh/home/news/2017-08-14-osu-idol-2017-auditions-now-open)
- [Introducing the osu! Coffee Hour](https://osu.ppy.sh/home/news/2017-08-15-introducing-the-osu-coffee-hour)
- [Yuzu's New Look Contest](https://osu.ppy.sh/home/news/2017-08-21-yuzus-new-look-contest)
- [New Featured Artist: OISHII](https://osu.ppy.sh/home/news/2017-08-30-new-featured-artist-oishii)

## Septembre

> [\*namirin](https://osu.ppy.sh/beatmaps/artists/18) a été annoncé en tant que nouveau Featured Artist. Mapping with rewards est de retour avec des récompenses liées aux musiques sélectionnées. osu!mania 4k World Cup s'est conclue par la victoire de la Corée du Sud face aux États-Unis. Les votes pour le nouveau look de Yuzu ont ouvert.

Reviewer: ```suggestion
[\*namirin](https://osu.ppy.sh/beatmaps/artists/18) a été annoncé en tant que nouveau featured artist. Mapping with rewards est de retour avec des récompenses liées aux musiques sélectionnées. L'osu!mania 4k World Cup s'est conclue par la victoire de la Corée du Sud face aux États-Unis. Les votes pour le nouveau look de Yuzu sont ouverts.
```

Liens :

- [New Featured Artist: Ben Briggs](https://osu.ppy.sh/home/news/2017-02-08-new-featured-artist-ben-briggs)
- [Release 0.0.3 · ppy/osu](https://github.com/ppy/osu/releases/tag/v0.0.3)

## Mars

Le Monthly Beatmapping Contest revolving pour osu!catch a commencé. [LukHash](https://osu.ppy.sh/beatmaps/artists/12) a été annoncé en tant que nouvel Featured Artist. Les votes pour Aspire 2017 ont commencé. Spotlights a été annoncé pour co-exister avec les chartes de classement. Le 9ème concours de fanart sur le thème printanier a ouvert.

Liens :

- [Monthly Beatmapping Contest March 2017: osu!catch](https://osu.ppy.sh/home/news/2017-03-01-monthly-beatmapping-contest-march-2017-osucatch)
- [New Featured Artist: LukHash](https://osu.ppy.sh/home/news/2017-03-03-new-featured-artist-lukhash)
- [Aspire 2017 - Voting Now Open!](https://osu.ppy.sh/home/news/2017-03-13-aspire-2017-voting-now-open)
- [Introducing to you: Spotlights](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights)
- [Fanart Contest \#9: Spring](https://osu.ppy.sh/home/news/2017-03-21-fanart-contest-9-spring)

## Avril

> 

Author: ```suggestion
```

Le Monthly Beatmapping Contest \#13 a conclu avec [RyoKazuka](https://osu.ppy.sh/users/6258586) à la première place. Aspire 2017 : Stage One s'est conclu par la première place de [ProfessionalBox](https://osu.ppy.sh/users/3250792). Les votes pour le concours de fanart printanier ont ouvert. Le 100ème(?) numéro du osu!weekly est sorti. La osu!taiko World Cup 2017 s'est conclue par la victoire du Japon sur la Taïwan. Avant que le mois d'avril se termine, le concours de fanart printanier ont permis à [Evaine](https://osu.ppy.sh/users/7295733) de prendre la première place.

Liens :

- [Monthly Beatmapping Contest \#13 Results](https://osu.ppy.sh/home/news/2017-04-02-monthly-beatmapping-contest-13-results)
- [Aspire 2017 Stage One Concludes](https://osu.ppy.sh/home/news/2017-04-03-aspire-2017-stage-one-concludes)
- [Spring Fan Art Contest Voting Now Open](https://osu.ppy.sh/home/news/2017-04-11-spring-fan-art-contest-voting-now-open)
- [osu!weekly \#100](https://osu.ppy.sh/home/news/2017-04-15-osuweekly-100)
- [osu!taiko World Cup 2017 Results](https://osu.ppy.sh/home/news/2017-04-22-osutaiko-world-cup-2017-results)

## Mai

> 

Author: ```suggestion
```
